src/util/get-cloud-or-server.ts

```diff
@@ -0,0 +1,4 @@
+import { GITHUB_CLOUD_API_BASEURL } from "utils/get-github-client-config";
+
+export const getCloudOrServerFromSubscription = (gitHubAppId: number | undefined) => gitHubAppId ? "server" : "cloud";
```

Comment:
/nit

Maybe `getCloudOrServerFromGitHubAppId` is more accurate than `getCloudOrServerFromSubscription`